internal/config/config.go

```diff
@@ -368,7 +368,9 @@ func Installer() (Install, error) {
 	// with Install we can just use nested keys.
 	// env variables can just make use of ";" to separat
 	artifacts := viper.GetStringSlice(ArtifactInstallArtifactsKey)
-	if len(artifacts) == 1 { // in this case it might come from the env
+	if len(artifacts) == 0 {
+		return Install{}, fmt.Errorf("cannot find any artifact to install")
+	} else if len(artifacts) == 1 { // in this case it might come from the env
```

Comment:
This is a good point, but not the right the right point to do it in the code in my opinion.
Here we are dealing with the configuration (config file + env variables), and you are saying that the install should fail if the user didn't configured any artifacts. However, artifacts can be passed as command line arguments to the install command. In that case, those are the artifacts to be used and install. So what I would do, is moving the check in the command itself, so that if neither the config or the args contain artifacts, the command fails. Does it make sense to you? Thank you :)

Reply:
See this for instance https://github.com/falcosecurity/falcoctl/blob/main/cmd/artifact/follow/follow.go#L208

Reply:
You are completely right! Makes a lot of sense! Thank you for the feedback! :D